mkdocs.yml

```diff
@@ -0,0 +1,68 @@
+site_name: fiware-cygnus
+site_url: https://fiware-cygnus.readthedocs.org
+repo_url: https://github.com/telefonicaid/fiware-cygnus.git
+site_description: Cygnus Documentation
+docs_dir: doc
+site_dir: html
+markdown_extensions: [toc,fenced_code]
+use_directory_urls: false
+theme: readthedocs
+extra_css: ["https://fiware.org/style/fiware_readthedocs.css"]
+pages:
+  - Home: index.md
+  - 'Contributing guidelines': 'contributing/contributing_guidelines.md'
```

Comment:
You are right, fixed in https://github.com/telefonicaid/fiware-cygnus/pull/949/commits/540563c64137c0bf10bcebaa9cf8f4dfe26093e1